### PR TITLE
Setting counting feature on projects section

### LINKED ISSUE #1629: Setting counting number feature on Our Project Section
**Is your feature request related to a problem? Please describe.**
Setting counting feature on [Our Project](https://sefglobal.org/) section 

**Describe the solution you'd like**
Update index.html file

**Additional context**

https://github.com/sef-global/sef-site/assets/142712365/dbc22452-2994-4efc-99b6-85d6959228ea

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by RamithaHeshan33**
## Purpose

The purpose of this PR is to fix #1629

## Goals

Setting the counting feature on our projects section

## Approach

update the index.html file

### Screenshots

Before fixing the issue

before

After fixing the issue

https://github.com/sef-global/sef-site/assets/142712365/b8e175b3-4ca5-4417-b8d9-e7cc9107d20f

### Preview Link

https://pr-1630-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -139,27 +135,22 @@ <h5 class="display-3 text-center project-stat">
...
-                            <h5 class="display-3 text-center project-stat">100+</h5>
+                                community created by SEF to showcase the work in various fields of Sri Lankan researchers.</p>
+                            <h5 class="display-3 text-center project-stat" id="irc-stat">100+</h5>
                             <p class="text-dark text-center stats">
                                 researchers of Sri Lankan origin representing the world's top universities
                             </p>
                         </div>
                     </a>
-		    <a href="onelive" class="card col-lg-4 shadow project-card">
+		            <a href="onelive" class="card col-lg-4 shadow project-card">
```
Comment (YoshithaRathnayake) [inline index.html]:
```suggestion
		    <a href="onelive" class="card col-lg-4 shadow project-card">
```
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by RamithaHeshan33** [comment]
@YoshithaRathnayake can you check if this PR is ok now please?

File: index.html [hunk 2]
```diff
@@ -139,27 +135,22 @@ <h5 class="display-3 text-center project-stat">
...
-                            <h5 class="display-3 text-center project-stat">100+</h5>
+                                community created by SEF to showcase the work in various fields of Sri Lankan researchers.</p>
+                            <h5 class="display-3 text-center project-stat" id="irc-stat">100+</h5>
                             <p class="text-dark text-center stats">
                                 researchers of Sri Lankan origin representing the world's top universities
                             </p>
                         </div>
                     </a>
-		    <a href="onelive" class="card col-lg-4 shadow project-card">
+		        <a href="onelive" class="card col-lg-4 shadow project-card">
```
Comment (YoshithaRathnayake) [inline index.html]: Can you remove this space? @RamithaHeshan33 its not still removed, it should be like the line 158

**Review by YoshithaRathnayake — DISMISSED** [review]

**Comment by RamithaHeshan33** [comment]
I think it's ok now. @YoshithaRathnayake

**Comment by YoshithaRathnayake** [comment]
> I think it's ok now. @YoshithaRathnayake

@anjula-sack will review rest! @RamithaHeshan33

**Comment by YoshithaRathnayake** [comment]
After this PR there is an error when click the scholarx card and again undo (return immediately before it redirects for the scholarx page) the scholarx logo of the card is hidden and the card's height is getting longer

**Review by anjula-sack — APPROVED** [review]
This is amazing! @RamithaHeshan33 Good job!

**Pull request merged**